dev-support/hbase_nightly_pseudo-distributed-test.sh

```diff
@@ -324,7 +324,7 @@ redirect_and_run "${working_dir}/hadoop_cluster_smoke" \
     "${hadoop_exec}" --config "${working_dir}/hbase-conf/" fs -ls -R /
 
 echo "Starting up HBase"
-HBASE_CONF_DIR="${working_dir}/hbase-conf/" "${component_install}/bin/start-hbase.sh"
+HBASE_CONF_DIR="${working_dir}/hbase-conf/" HBASE_LOG_DIR="${working_dir}" "${component_install}/bin/start-hbase.sh"
```

Comment:
nit: should this not be ${working _dir}/logs?

Reply:
We just output the related logs to working_dir

https://ci-hbase.apache.org/job/HBase%20Nightly/job/branch-2/972/artifact/output-integration/hadoop-3/

Reply:
Ah cool. i'm fine with this.